docs/source/modules.rst

```diff
@@ -14,6 +14,15 @@ analyses/buildingdamage
     :members:
 ..  autoclass:: buildingdamage.buildingutil.BuildingUtil
     :members:
+..  deprecated:: 1.19.0
+    This class will be deprecated soon. Use :class:`buildingstructuraldamage.BuildingStructuralDamage` instead.
+
+analyses/buildingstructuraldamage
+=======================
```

Comment:
I am very sorry for this because I am not very sure, but sphinx makes rendering error that doesn't actually giving any error message but doesn't show the part when the number of "=" doesn't match with the title above. So could you match the length of "=" as the same  for the number of characters in line 20?

Reply:
Thanks for finding this out! I just pushed the change commit